rskj-core/src/test/java/co/rsk/mine/MinerManagerTest.java

```diff
@@ -358,4 +358,11 @@ public boolean isValid(Block block) {
             return true;
         }
     }
+
+    private static class RskImplForTest extends RskImpl {
```

Comment:
Nit: Maybe create a method instead?

Reply:
What's _nit_?

Reply:
https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak

Reply:
We need this as a class b/c the test creates anonymous classes extending this one and overrides some methods